internal/pkg/cli/env_deploy_test.go

```diff
@@ -215,6 +225,7 @@ func TestDeployEnvOpts_Execute(t *testing.T) {
 				m.deployer.EXPECT().UploadArtifacts().Return(map[string]string{
 					"mockResource": "mockURL",
 				}, nil)
+				m.describer.EXPECT().ValidateCFServiceDomainAliases().Return(nil).Times(0)
```

Comment:
i'm _pretty sure_ omitting this does the same thing as `.Times(0)` (the test will fail if an unexpected function is called)